packages/sdk/src/client/edition/create.ts

```diff
@@ -96,7 +90,7 @@ export async function createEdition(
      */
     switch (mintConfig.mintType) {
       case 'RangeEdition': {
```

Comment:
create will use v2 minters after this change

Reply:
this is technically a breaking change if the older contract addresses would break

Reply:
won't break, they have the same function signature